torchtext/prototype/models/t5/modules.py

```diff
@@ -448,7 +448,7 @@ def _relative_position_bucket(
         Returns:
             a Tensor with the same shape as relative_position, containing int32 values in the range [0, num_buckets)
         """
-        relative_buckets = torch.zeros(relative_position.shape, dtype=torch.long)
+        relative_buckets = torch.zeros(relative_position.shape, dtype=torch.long, device=relative_position.device)
```

Comment:
can we change to `device=device` for simplicity?

Reply:
Do you mean `device=self.device`?